struct_env/struct_corr_env.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
```

Comment:
This file should be removed before accepting the PR.
(once everything is done)

Reply:
Agreed! Let's remove duplicated code.